### PR TITLE
Add advisory for vrana/adminer file disclosure vulnerability

### PR DESCRIPTION
**Pull request by aTastyCookie**
- https://sansec.io/research/adminer-4.6.2-file-disclosure-vulnerability
- https://www.reddit.com/r/programming/comments/ahspfv/mysql_client_allows_mysql_server_to_request_any/

### CODE REVIEW

File: vrana/adminer/2019-01-20.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     Adminer script versions from 4.3.1 up to 4.6.2 contains file disclosure vulnerability
+link:      https://sansec.io/research/adminer-4.6.2-file-disclosure-vulnerability
+reference: composer://vrana/adminer
+cve:       ~
+branches:
+   master:
+       time:     2019-01-20 00:00:00
+       versions: ['>=4.3.1', '<4.6.3']
```
Comment (stof) [inline vrana/adminer/2019-01-20.yaml:8]: The post you link does not say that versions before 4.3.1 are unaffected, but that the researcher stopped testing them. I suggest removing the lower bound here.

File: vrana/adminer/2019-01-20.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title:     Adminer script versions from 4.3.1 up to 4.6.2 contains file disclosure vulnerability
+link:      https://sansec.io/research/adminer-4.6.2-file-disclosure-vulnerability
+reference: composer://vrana/adminer
+cve:       ~
+branches:
+   master:
+       time:     2019-01-20 00:00:00
```
Comment (stof) [inline vrana/adminer/2019-01-20.yaml:7]: 4.6.3 has been released on 2018-06-28

File: vrana/adminer/2019-01-20.yaml (hunk 1 above)
Comment (aTastyCookie) [reply vrana/adminer/2019-01-20.yaml:8]: Fixed, thaks.

File: vrana/adminer/2019-01-20.yaml (hunk 2 above)
Comment (aTastyCookie) [reply vrana/adminer/2019-01-20.yaml:7]: Also fixed, thank you!

**Review by stof — APPROVED** [review]
Looks good to me (but I don't have merge rights here)

**Pull request merged**